packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -76,3 +81,30 @@ Attribute groups can be imported by their ARN via the `AttributeGroup.fromAttrib
 const importedAttributeGroup = appreg.AttributeGroup.fromAttributeGroupArn(this, 'MyImportedAttrGroup',
   'arn:aws:servicecatalog:us-east-1:012345678910:/attribute-groups/0aqmvxvgmry0ecc4mjhwypun6i');
 ```
+
+## Associations
+
+You associate your appregistry application with attribute groups and resources.
+Resources are CloudFormation stacks that you can associate to application to group relevant
```

Comment:
```suggestion
Resources are CloudFormation stacks that you can associate to an application to group relevant
```

Reply:
I think with verbiage this should be "with", fixed.